srcpkgs/mailnag/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'mailnag'
+pkgname=mailnag
+version=2.2.0
+revision=1
+build_style=python3-module
+hostmakedepends="python3 gettext"
+# a libsecret client (GNOME Keyring, Kwallet...) is needed
+# a libnotify client is needed (if you want to use the libnotify extention)
+# You should also have a working syslog (https://docs.voidlinux.org/config/services/logging.html) if you do not want error messages about syslog
+depends="python3 python3-gobject python3-dbus python3-xdg libsecret gtk+3 $(vopt_if sound gst-plugins-base1) $(vopt_if notify libnotify)"
+build_options="notify sound"
+desc_option_notify="Adds support for libnotify plugin"
+desc_option_sound="Adds support for sound notification plugin"
+build_options_default="notify sound"
+make_install_args="--optimize=1 --skip-build"
```

Comment:
bytecode optimizing is already handled by build style, --skip-build isn't neede either

Reply:
Fixed.